AIPscan/API/namespace_data.py

```diff
@@ -25,42 +25,42 @@
 
 
 @api.route("/aip-overview/<storage_service_id>")
-class FMTList(Resource):
+class AIPList(Resource):
     @api.doc(
         "list_formats",
         params={
             fields.FIELD_ORIGINAL_FILES: {
-                "description": "Return data for original files or copies",
+                "description": "Return data on original files or preservation derivatives",
                 "in": "query",
                 "type": "bool",
             }
         },
     )
     def get(self, storage_service_id):
-        """List PUIDs and AIPs containing examples of them"""
+        """Return data on AIPs and the file formats they contain."""
         original_files = parse_bool(request.args.get(fields.FIELD_ORIGINAL_FILES, True))
-        aip_data = data.aip_overview(
+        aip_data = data.aip_overview_two(
```

Comment:
This predates your specific work, but can we rename "aip_overview_two" to be more descriptive, rather than just having two similarly-named `aip_overview` and `aip_overview_two` fields? Like `aip_file_formats_overview` or something that gives more context.